src/config/dynamic-theme-fixes.config

```
@@ -20829,6 +20829,16 @@ div.banner
 
 ================================
 
+vnexpress.net
+
+INVERT
+.item span
```

`.item span` sounds too general.

```suggestion
#vibeji-ham
```